manager.py

```diff
@@ -0,0 +1,111 @@
+"""Manager class to handle all the inputs given by user in Main class"""
+from jsonhandler import *
+from event import *
+from datetime import *
+import uuid
+
+
+class Manager(object):
+    """
+        Attribute:
+            json_instance: Instance of JsonHandler class
+        Methods:
+            add_event(),
+            remove_event_by_id(id),
+            update_event_by_id(event_id, field_name, field_value),
+            search_event_by_id(event_id),
+            today_upcoming_and_completed_events(),
+            list_events_by_date_and_city(),
+            list_events_in_date_range().
+    """
+
+    def __init__(self):
+        self.json_instance = JsonHandler()
+
+    def add_event(self, event_instance):
+        id = uuid.uuid4()
+        self.json_instance.dump_file({str(id): event_instance.__dict__})
```

Comment:
Instead of event_instance.__dict__ , expose a event.toJSON() method which will return the event as a dictionary. Thus you will have more control over it and can reuse